src/api/conflictResolution.ts

```diff
@@ -312,20 +312,20 @@ export const sortTimestampedDoc = <
 }
 
 export const timestampSubVersion = (timestampRevs: TimestampRev[]): string => {
+  const subVersionLength = 16
+  const subVersionFactor = '1' + '0'.padEnd(subVersionLength, '0')
```

Comment:
Move these to all-cap constants at the top of the file to signify their importance as algorithm constants.